### ISSUE TITLE
Problem under Recent Repositories with Ampersand (&) in Windows Path

### ISSUE THREAD

**Issue by Duchtel** [gitextensions/gitextensions#7042]
## Current behaviour
The repository path will not be displayed if the (Windows) path contains an &. Only the branch is shown.
Start > Recent Repositories shows the path but substitutes the & with a space (e.g. "\~\Stuff\A   B\C" instead of "~\Stuff\A & B\C".

## Expected behaviour
The full path including the & should be shown.

## Steps to reproduce
This can be reproduced by cloning any repository in a folder whose path contains a & (e.g. C:\Users\Duchtel\Stuff\A & B\C).

## Environment

- GitExtensions version: 3.1.16049
- GIT version: 2.20.0
- OS version: Windows 10 (1809)
- .NET version: 

## Diagnostics
GitExtensions.settings seems to escape the RepositoryHistory …

![GitExtensions settings](https://user-images.githubusercontent.com/54449128/63608882-f575f400-c5a3-11e9-8267-37d845d07e51.png)

**Comment by mstv** [comment]
It is only a display issue. The repo can be opened successully.
For the Recent Repo List, a menu is used as GUI control. `&` marks the character for the hotkey for keyboard menu navigation/activation. Hit the`Alt` key and windows will underline the following space.

**Comment by Duchtel** [comment]
Yes ... this a cosmetic/display issue only but the & should still be properly escaped.
